cloud/terraform/machineactuator.go

```diff
@@ -406,7 +441,7 @@ func (tf *TerraformClient) updateMasterInPlace(goalMachine *clusterv1.Machine) e
 	// Control plane upgrade
 	if goalMachineControlPlaneVersion != currentControlPlaneVersion {
 		// Pull the kudeadm for target version K8s.
-		cmd := fmt.Sprintf("curl -sSL https://dl.k8s.io/release/v%s/bin/linux/amd64/kubeadm | sudo tee /usr/bin/kubeadm > /dev/null; " +
+		cmd := fmt.Sprintf("curl -sSL https://dl.k8s.io/release/v%s/bin/linux/amd64/kubeadm | sudo tee /usr/bin/kubeadm > /dev/null; "+
```

Comment:
Why the change here?

Reply:
gofmt. My editor runs it on every save and it does this.